frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -516,8 +529,8 @@ public static function apiDetails(\OmegaUp\Request $r) {
         $r->ensureIdentity();
 
         $r->ensureInt('qualitynomination_id');
-        /** @var null|array{status: string, nominator: array{username: string}, problem: array{alias: string}, contents: array{statements: array<string, string>}} $response */
-        $response = \OmegaUp\DAO\QualityNominations::getByID($r['qualitynomination_id']);
+        /** @var null|array{status: string, nominator: array{username: string}, problem: array{alias: string}, contents: array{statements: array<string, string>}, original_contents: array{tags: array{name: string, autogenerated: bool}[]}} $response */
+        $response = \OmegaUp\DAO\QualityNominations::getByID((int)$r['qualitynomination_id']);
```

Comment:
```suggestion
        $response = \OmegaUp\DAO\QualityNominations::getByID(
            intval($r['qualitynomination_id'])
        );
```

se prefiere el uso de `intval()`/`boolval()`/`strval()` sobre el casting. igual en los demás lugares donde usas `(int)` en este cambio.